packages/python/plotly/_plotly_utils/tests/validators/test_angle_validator.py

```diff
@@ -51,6 +52,14 @@ def test_aok_acceptance(val, validator_aok):
     assert np.array_equal(validator_aok.validate_coerce(np.array(val)), np.array(val))
 
 
+# Test base64 array
+def test_aok_base64_array(validator_aok):
+    val = b64(np.array([1, 2, 3], dtype="int64"))
```

Comment:
Great that you tested `int64` i.e. [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) in JavaScript.
Since we decided not to handle BigInts on the plotly.js side (for now) `int64` and `uint64` are passed as other types in regards with the minimum and maximum values used in the array. 
See here: https://github.com/plotly/plotly.py/blob/644992f0fe4c3f869ba45e5627e65bf1f6537f7c/packages/python/plotly/_plotly_utils/basevalidators.py#L89-L112

It would be great if you add tests for each of those cases.
Thank you!

Reply:
The `angle` valType is restricted to small integers between -360 and 360. So let's add those tests to `integer` valType instead.